TechTable/index.php

```diff
@@ -0,0 +1,113 @@
+<?php
```

Comment:
You're not using php here, so remove the php block.

Also if it doesn't break things, rename this file to index.html

Reply:
The php block has been removed. 

However, the web application doesn't work upon renaming the file to index.html.